docs-src/app-dev-context/custom-payload-conversion.md

```diff
@@ -0,0 +1,19 @@
+---
+id: custom-payload-conversion
+title: How to use custom payload conversion
+description: Create your custom `PayloadConverter` and set it on a `DataConverter` in your Client options.
+sidebar_label: Custom payload conversion
+tags:
+  - guide-context
+---
+
+Most SDKs provide a [`PayloadConverter`](/concepts/what-is-a-payload-converter) that can be customized to convert values in custom data types to bytes and back.
+
+Creating a custom Payload Converter is optional.
+It's needed only if the default Data Converter does not support your custom values.
+
+Create your [custom `PayloadConverter`](/concepts/what-is-a-payload-converter#custom-payload-conversion) and set it on a `DataConverter` in your Client options.
+
+The order in which your encoding Payload Converters are applied depend on the order given to the Data Converter.
+You can set multiple custom encoding Payload Converters to run your conversions.
+When converting from a value to a payload, only the first one that handles the data type of the value will apply.
```

Comment:
If we even put this detail here, we should enrich it a little. "the first one that handles the data type of the value" is a little sudden and not even correct because the user can actually program much more complex logic than just a datatype or class-based one.
Something like

"
Every `PayloadConverter` can handle the input or pass it forward without changing it. When converting from a value to a payload, the first `PayloadConverter` that handles the input will apply.
"

Reply:
Updated to 
```
The order in which your encoding Payload Converters are applied depend on the order given to the Data Converter.
You can set multiple custom encoding Payload Converters to run your conversions.
When the Data Converter receives a value for conversion, it passes through each `PayloadConverter` in sequence until the converter that handles the data type does the conversion.
```